docs/io/configuration/components/models/converters/arepo_to_tardis.ipynb

```diff
@@ -390,6 +390,46 @@
     "\n",
```

Comment:
<p>...although <strong>they</strong> <strong>cannot currently</strong> be used... ...<strong>functionality</strong>...</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2092/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>